.gitignore

```diff
@@ -4,7 +4,7 @@
 .gdb_history
 .libs/
 .*.swp
-Makefile
+Makefile$
```

Comment:
what is the reason for this? no mention of it in the commit message

Reply:
I am sorry about the missing message, it is a bugfix which prevents `Makefile.am` files to be git-ignored.